src/main.cpp

```diff
@@ -25,7 +25,11 @@
 #include "log.h"
 #include "r_util.h"
 #include "host.h"
+
+#ifdef _WIN32
```

Comment:
I'm thinking maybe instead of repeating this everywhere, put the guards inside `Rapi.h` itself, around its entire content?